charts/px-central/templates/px-lighthouse/px-central-ui/pxcentral-lighthouse.yaml

```diff
@@ -148,6 +148,10 @@ spec:
           - name: SSL_CERT_DIR
             value: /tmp/certs
           {{- end }}
+          {{- if eq .Release.IsInstall true }}
+          - name: QUERY_MONGO_DB
+            value: "false"        
```

Comment:
lets create a task to remove this for all the further releases. Else I guess a customer who installed 2.7.0 fresh, but upgrades to 2.7.1 will end-up querying mongodb, which is not needed.